src/sentry/features/temporary.py

```diff
@@ -628,3 +628,11 @@ def register_temporary_features(manager: FeatureManager):
         FeatureHandlerStrategy.FLAGPOLE,
         api_expose=True,
     )
+
+    # Feature Flags Features.
+    manager.add(
+        "organizations:feature-flag-autocomplete",
```

Comment:
```suggestion
        "organizations:feature-flag-search",
```
autocomplete is just the FE

Reply:
Let's share the flag.